test_app/urls.py

```diff
@@ -2,9 +2,11 @@
 from django.urls import include, path, re_path
 
 from ansible_base.lib.dynamic_config.dynamic_urls import api_urls, api_version_urls, root_urls
+from test_app.views import router as user_router
 
 urlpatterns = [
     path('api/v1/', include(api_version_urls)),
+    path('api/v1/', include(user_router.urls)),
```

Comment:
Hmmm.... we don't say this in the documentation but we had indirectly said before that a service should puil in all of the URLs from DAB first before adding any of their own. That way if something got hosed up in DAB and `api_urls` contained an entry like api/v1 it wouldn't be able to overwrite the services urls. So this line should be after `path('', include(root_urls)),` and we should probably formally note this better in Installation.md.

Reply:
Also, we should think about how we want to use routers. In other services that use them we have a routers.py file at the root instead of in the views. It would be nice to be consistent across services.

Reply:
> In other services that use them we have a routers.py file at the root instead of in the views. It would be nice to be consistent across services.

I don't know, https://github.com/ansible/aap-gateway/blob/devel/aap_gateway_api/urls.py looks just as messy as what we have here. Introducing a `routers.py` won't change how it includes views and includes routers.

Yes to the prior thing, to have test_app model the intended use. Pushed.